Update file-sync-choose-cloud-tiering-policies.md (#127746)
Azure/azure-content: articles/storage/file-sync/file-sync-choose-cloud-tiering-policies.md

|1 MiB (1,048,576 bytes) | 2 MiB |
|2 MiB (2,097,152 bytes) | 4 MiB |

> The volume cluster size can be found by executing the command `fsutil fsinfo ntfsinfo volumedriveletter:` from an administrative command prompt. The **Bytes Per Cluster** field will display the volume cluster size in bytes with the value in kilobytes shown in parentheses.

Reviewer: ```suggestion
The volume cluster size can be found by executing the command `fsutil fsinfo ntfsinfo volumedriveletter:` from an administrative command prompt. The **Bytes Per Cluster** field displays the volume cluster size in bytes, and the value in kilobytes is shown in parentheses.
```
Small edits for Microsoft style.
